examples/.rayproject/project.yaml

```diff
@@ -0,0 +1,23 @@
+# This file is generated by `ray project create`.
+
+name: ray-tutorial-examples
+
+description: "The Ray tutorial examples"
+tags: ["ray-tutorial", "examples", "newsreader", "map-reduce", "modin"]
+
+cluster: .rayproject/cluster.yaml
+
+environment:
+  requirements: .rayproject/requirements.txt
+
+commands:
+  - name: tutorial
+    help: "Start a jupyter notebook with the tutorials"
+    command: jupyter notebook
```

Comment:
`--port 8889` otherwise the default port will be used.

Reply:
Ok, changing it (it was the default port for me)

Reply:
also `--ip="0.0.0.0"` i think it binds to localhost by default and not opened to otherwise places

Reply:
actually this was using port forwarding to connect (because with the autoscaler no ports are open by default), so I canged the `--ip=` part back :)